frontend/messages/en/PrivacyPolicy.json

```diff
@@ -54,34 +54,28 @@
         {
           "key": "usageMetrics",
           "title": "Usage Metrics",
-          "description": "Days logged in, training sessions, AI tokens used, ...",
-          "purposes": ["Track user progress", "Mark achievements", "Provide app usage metrics"],
+          "description": "Total number of sessions, total training time, average score, current login streak",
+          "purposes": [
+            "Track user progress",
+            "Mark achievements",
+            "Provide app usage metrics as a whole for administrators"
+          ],
           "storage": [
             "Collected at various points as the user interacts with the app",
             "Retained until account deletion"
           ]
         }
       ]
     },
-    "security": {
-      "title": "Security & Data Protection",
-      "euServers": {
-        "title": "EU-Based Servers",
-        "description": "All data is stored and processed on servers located in the European Union"
-      },
-      "encryption": {
-        "title": "Encryption",
-        "description": "Data is encrypted both in-transit and at-rest for maximum security"
-      },
-      "externalServices": {
-        "title": "External Service Providers",
-        "services": [
-          "Vercel - EU servers and global CDN",
-          "Supabase - EU servers",
-          "Fly.io - EU servers",
-          "GCP - EU servers"
-        ]
-      }
+    "externalServices": {
+      "title": "External Service Providers",
```

Comment:
Let's change the title to "data processors" in English and "Auftragsverarbeiter" in German to conform to GDPR nomenclature.